src/oracle.c

```diff
@@ -1102,13 +1103,18 @@ main(int argc, char **argv)
 			opt_pcr_policy = optarg;
 			break;
 		case OPT_KEY_FORMAT:
-			opt_key_format = optarg;
+			warning("Detected --key-format option; please use --target-platform instead\n");
+			opt_target_platform = optarg;
```

Comment:
We are in 0.5, so a bit far of 1.0. Maybe is not so bad to drop this part and have a hard fail instead of a deprecation warning?

Reply:
Well, the code is in use on ALP, so we need a graceful transition. But I can omit the warning of course :-)

Reply:
Maybe we can set the target version to 0.6 to drop "--key-format", so that we could have some time to update fde-tools and add a hard requirement for pcr-oracle version.